pom.xml

```diff
@@ -160,7 +160,7 @@
         <dep.packaging.version>${dep.airlift.version}</dep.packaging.version>
         <dep.takari.version>2.1.1</dep.takari.version>
         <dep.aws-sdk.version>1.12.577</dep.aws-sdk.version>
-        <dep.aws-sdk-v2.version>2.21.4</dep.aws-sdk-v2.version>
+        <dep.aws-sdk-v2.version>2.21.11</dep.aws-sdk-v2.version>
```

Comment:
Let's remove this property since it's only used in one place for the BOM import

Reply:
I find it easier to work with the properties but maybe it doesn't matter.